wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift

```diff
@@ -97,12 +97,14 @@ final class ClientListViewController: UIViewController,
         return nil
     }
 
-    required init(clientsList: [UserClient]?,
-                  selfClient: UserClient? = ZMUserSession.shared()?.selfUserClient,
-                  credentials: ZMEmailCredentials? = .none,
-                  detailedView: Bool = false,
-                  showTemporary: Bool = true,
-                  showLegalHold: Bool = true) {
+    required init(
+        clientsList: [UserClient]?,
+        selfClient: UserClient? = ZMUserSession.shared()?.selfUserClient,
```

Comment:
can we make this non optional? if we don't have a selfUserClient we should not present this screen, no?

Reply:
Not in the scope of this PR, But I will make a note

Reply:
[WPB-5734](https://wearezeta.atlassian.net/browse/WPB-5734) Made a note here